scripts/abi_artifact_generator/templates/printers.inc.jinja_template

```diff
@@ -0,0 +1,199 @@
+/*#-
+This template file is distributed under the MIT License. See LICENSE.md for details.
+#*/
+
+/*(- generation_notice )*/
+
+#include "../common.h"
+#include "../functions.h"
+#include "../types.h"
+
+_Static_assert(sizeof(uint8_t) == 1, "A type with size == 1 is required.");
+#define MAKE_PRINT_HELPER(TYPE, POINTER, RESULT) \
```

Comment:
This thread is a reminder about a carry-over thread:
https://github.com/revng/revng-qa/pull/17#discussion_r1354322243

Reply:
> Hoist in one of the headers above or in a new one?
Same with functions at the bottom.

This was streamlined somewhat (not as many macros :tada:)

But still, consider that this is only used here, you sure you want a header for it?